.github/workflows/release-github.yml

```diff
@@ -37,3 +37,12 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  release-to-winget:
+    needs: [ build ]
+    runs-on: windows-latest
+    steps:
+      - name: Release to Winget
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: ORASProject.ORAS
+          token: ${{ secrets.WINGET_GITHUB_TOKEN }}
```

Comment:
This step should be in a different workflow file since this file is intended for releasing to the GitHub releases.

Reply:
This can be done by [`on.workflow_run`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_runbranchesbranches-ignore)